app/models/katello/glue/elastic_search/product.rb

```diff
@@ -1,37 +1,5 @@
 module Katello
   module Glue::ElasticSearch::Product
```

Comment:
we could move them to the base model, but i'd rather leave the two other search methods in an es glue module